src/sass/extends/_tooltips.scss

```diff
@@ -1,123 +1,121 @@
-.uds-tooltip {
-  // Turn tooltip icon blue on hover
-  &:hover {
-    .fa-stack .fa-circle {
-      color: $uds-color-font-light-info;
-    }
-  }
+div.uds-tooltip-container {
+  display: inline-block;
+  position: relative;
+}
 
-  // Display tooltip text on icon hover
-  &:hover &-text {
-    visibility: visible;
+button.uds-tooltip {
```

Comment:
- This is a relatively new element for UDS-Bootstrap.
- Looks like they needed to make multiple accessibility fixes to the initial creation. Looks solid now.
- The options for implementing a background color for the element rely on classes that are specifically written for this element. I'd like to figure out how we can instead use the background section block to do the same thing before we implement this as a block. 
- ... and that's if we decide to roll this out as a block. A tooltip is inherently an inline kind of element which may not behave or "fit" well within a block-centered design. 

OK to merge. We'll keep our eye on it.